README.md

```diff
@@ -1,13 +1,10 @@
-# Project Name
-
-Replace this readme with your own information about your project.
-
-Start by briefly describing the assignment in a sentence or two. Keep it short and to the point.
+# Project chat bot
+The aim of this project was to build a chat bot using Javascript.
 
 ## The problem
+The website was already structured with HTML and CSS. We used Javascript to build a chat bot that is answering questions using functions and conditional statements. We started with a chain of if/else statements, but later developed this into several functions passing arguments.
 
-Describe how you approached to problem, and what tools and techniques you used to solve it. How did you plan? What technologies did you use? If you had more time, what would be next?
+If we would have more time, we would develop the ability to accept several different answers from user.
 
 ## View it live
-
-Have you deployed your project somewhere? Be sure to include the link to the deployed project so that the viewer can click around and see what it's all about.
+https://serene-mcclintock-46e815.netlify.app/
```

Comment:
README file is a very special place in your repo. You should try to 'sell' your project here. 
Use it to tell your users what your application does, how to use it and eventually what next functionalities can be added in the future. 
Make it shine because this is cover of your project and you need to make your visitors stop for a while and look deeper to your code.